packages/components/src/components/Calculator/CalculatorResult.tsx

```diff
@@ -21,6 +21,16 @@ type Props = {
   autorun: boolean;
 };
 
+const MemoizedValueResultViewer = memo(function MemoizedValueResultViewer({
```

Comment:
No need for extra component here, you could wrap `ValueResultViewer` definition in `memo`.

(it's almost never a problem if a component is wrapped in `memo`)

Reply:
Changed